### PR TITLE
Enable D102 rule for pydocstyle

### PR DESCRIPTION
**Pull request by AntonKarabaza**
Add docstrings for public methods.
Modify .pydocstyle to enable D102 rule but ignore D212 rule to
resolve conflicting rules.

Affects #25

### CODE REVIEW

File: .pydocstyle [hunk 1]
```diff
@@ -1,2 +1,8 @@
 [pydocstyle]
-ignore = D1
\ No newline at end of file
+# D100: Missing docstring in public module
+# D101: Missing docstring in public class
+# D103: Missing docstring in public function
+# D104: Missing docstring in public package
+# D212: Multi-line docstring summary should start at the first line
+
+ignore = D100,D101,D103,D104,D212
```
Comment (ivan-podorozhnyi) [inline .pydocstyle]: @AntonKarabaza Please add an empty line at the end of the file.

File: demo_python_at/commons/message.py [hunk 2]
```diff
@@ -4,42 +4,91 @@
...
+
+        :param message: data in string format
+        """
         self._message = message
 
     def exist(self) -> bool:
+        """
+        Verify that message length greater than 0.
+
+        :return: True if message length greater than 0
```
Comment (ivan-podorozhnyi) [inline demo_python_at/commons/message.py]: @AntonKarabaza Please correct grammar mistakes. Its right to say "`is` greater than".

**Review by ivan-podorozhnyi — CHANGES_REQUESTED** [review]
@AntonKarabaza Please look at my comments.

**Comment by AntonKarabaza** [comment]
@ivan-podorozhnyi 
Empty line added in the end of `.pydocstyle`.
Grammar mistakes corrected in `demo_python_at/commons/message.py`.

**Comment by ivan-podorozhnyi** [comment]
@AntonKarabaza Thank you!

**Review by ivan-podorozhnyi — APPROVED** [review]

**Comment by extsoft** [comment]
@AntonKarabaza please fix conflicts.

**Comment by AntonKarabaza** [comment]
@extsoft conflicts resolved.

**Review by extsoft — APPROVED** [review]

**Pull request merged**